src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -189,6 +190,160 @@ public function sendPhpResponse($exception)
         echo $this->decorate($this->getContent($exception), $this->getStylesheet($exception));
     }
 
+    /**
+     * Gets the content associated with the given exception.
+     *
+     * @param \Exception|FlattenException $exception An \Exception or FlattenException instance
+     * @param string                      $format    The request format (html, json, xml, txt)
+     *
+     * @return string The formatted content as a string
+     */
+    public function getFormattedContent($exception, string $format): string
+    {
+        switch ($format) {
+            case 'json':
+                return $this->getJson($exception);
+            case 'xml':
+            case 'rdf':
+                return $this->getXml($exception);
+            case 'txt':
+                return $this->getTxt($exception);
+            default:
+                return $this->getHtml($exception);
+        }
+    }
+
+    /**
+     * Gets the JSON content associated with the given exception.
+     *
+     * @param \Exception|FlattenException $exception An \Exception or FlattenException instance
+     *
+     * @return string The JSON content as a string
+     */
+    public function getJson($exception): string
+    {
+        if (!$exception instanceof FlattenException) {
+            $exception = FlattenException::create($exception);
+        }
+
+        if (404 === $statusCode = $exception->getStatusCode()) {
+            $title = 'Not Found';
+        } elseif (class_exists(Response::class) && isset(Response::$statusTexts[$statusCode])) {
+            $title = Response::$statusTexts[$statusCode];
+        } else {
+            $title = 'Internal Server Error';
```

Comment:
so a 403 error without `symfony/http-foundation` installed would lead to this title/message? Might be a bit confusing :smile: